src/plopp/graphics/colormapper.py

```diff
@@ -55,6 +55,8 @@ class ColorMapper:
     cax:
         The axes to use for the colorbar. If none are supplied, the ColorMapper will
         create its own axes.
+    cbar:
+        Create a colorbar if ``True``.
```

Comment:
This argument has no effect if `cax` is set. Is this intentional? If so, please document.

Reply:
Not intentional, i'll fix.